app/uk/gov/hmrc/helptosave/models/CallingCodes.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosave.models
+
+import cats.instances.int._
+import cats.syntax.eq._
+
+import scala.collection.Map
+import scala.io.Source
+
+object CallingCodes {
+
+  private val callingCodes: Map[Int, String] = {
+    var codes = Map[Int, String]()
+    val content = Source.fromInputStream(getClass.getResourceAsStream("/resources/callingcodes.txt")).getLines()
+    content.foreach {
```

Comment:
you could do a `foldLeft` here instead of having a mutable `var` and updating it in a loop. Then you could have something like this:
```scala
content.foldLeft(Map.empty[Int,String]){ case (acc, curr) =>
  curr.split("-").toList match {
    case key :: value :: Nil => acc.updated(key.toInt, value)
    case _ => acc
  }
}
```
Note in the above I've avoided doing a `get` on the array and got the key-value pair via pattern matching